src/index.js

```diff
@@ -87,14 +87,17 @@ exports.resolve = (source, file, config = {}, babelrc = '.babelrc') => {
 
     const rootPathConfig = optsArray.map((item = {}) => ({
         rootPathPrefix: isString(item.rootPathPrefix) ? item.rootPathPrefix : '~',
-        rootPathSuffix: isString(item.rootPathSuffix) ? item.rootPathSuffix.replace(/^(\/)|(\/)$/g, '') : ''
+        rootPathSuffix: isString(item.rootPathSuffix) ? item.rootPathSuffix.replace(/^(\/)|(\/)$/g, '') : '',
+        ...item,
     }));
 
     let transformedSource = source;
+    let resolverConfig = {};
     for (let i = 0; i < rootPathConfig.length; i += 1) {
-        const option = rootPathConfig[i];
-        const prefix = option.rootPathPrefix;
-        const suffix = option.rootPathSuffix;
+        // The remaining configs will be sent to node.resolver to deal with special cases.
+        // eg. Adding .jsx to the extension list.
+        const { rootPathPrefix: prefix, rootPathSuffix: suffix, ...option } = rootPathConfig[i];
+        resolverConfig = option;
```

Comment:
you should define the default configuration as defined here:
https://github.com/unconfident/eslint-import-resolver-babel-plugin-root-import/blob/master/src/index.js#L89-L90

Otherwise it'll fail when using the plugin without options:
```js
{
  "plugins": [
    "babel-plugin-root-import",
    ...
  ]
}

```